src/modules/menu/components/header/HeaderLink.vue

```diff
@@ -0,0 +1,44 @@
+<template>
+    <button
+        class="btn btn-xs btn-link m-0 px-1 custom-text-decoration"
+        :class="{
+            'text-black': !selected,
+            'text-primary': selected,
+        }"
```

Comment:
I would leave the logic of selection outside of this component, this is specific to language. It is also not intuitive to set `selected=true` for the feedback button just to have it in the primary color. So this class should be by default `black` and with a prop could be changed in `primary`, `secondary`, `danger`, ...
We need to keep in mind that we might have in future other LinkButton with other colors.